packages/api/prisma/schema.prisma

```diff
@@ -14,29 +14,54 @@ datasource db {
 }
 
 model Book {
-  id      Int       @id
-  name    String
-  search  String    @unique // This gives us a human readable key to look up books from URL params.
-  Chapter Chapter[]
-}
-
-model Chapter {
   id     Int     @id
-  number Int
-  book   Book    @relation(fields: [bookId], references: [id])
-  bookId Int
+  name   String
+  search String  @unique // This gives us a human readable key to look up books from URL params.
   Verse  Verse[]
-
-  @@unique([bookId, number])
 }
 
 model Verse {
-  id        Int     @id
-  number    Int
-  chapter   Chapter @relation(fields: [chapterId], references: [id])
-  chapterId Int
+  id      Int    @id
+  number  Int
+  chapter Int
+  book    Book   @relation(fields: [bookId], references: [id])
+  bookId  Int
+  Word    Word[]
+
+  @@unique([chapter, number])
```

Comment:
The chapter table isn't providing much value, so we can remove a join by just putting the chapter number in the verse